scripts/utils/generate_package_wiki.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import os
+import pathlib
+import sys
+import xml.etree.ElementTree as ET
+from collections import defaultdict
+
+packagesByCategory = defaultdict(str)
+
+
+def sort_write_wiki_content(file_path):
+    """
+    iterates over each category from the dictionary packagesByCategory and
+    writes the Markdown formatted string that contains
+    - header with the category name
+    - table with two columns: Package and Description of each of the packages"""
```

Comment:
Use Google recommended style to document Python functions in this and the other functions: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#383-functions-and-methods